numbat/modules/units/misc.nbt

```diff
@@ -115,6 +115,11 @@ unit swimmingpool: Volume = 50 m × 25 m × 2 m # Olympic-size swimming pool (FR
 @aliases(rackunits, RU: short, U: short)
 unit rackunit: Length = 0.04445 meter
 
+@metric_prefixes
+@aliases(darcys, darcies, D: short)
```

Comment:
I would prefer if we keep the one-letter constant `D` free for now. Users can still set `unit D = darcy` (potentially with `@metric_prefixes`) if they want.
```suggestion
@aliases(darcys, darcies)
```